pegasus/migrations/105_add_contacts_rollup_table.rb

```diff
@@ -0,0 +1,29 @@
+Sequel.migration do
+  change do
+    create_table(:contacts_rollup, charset: 'utf8') do
+      primary_key :id, unsigned: true, null: false
+      String :email
+      Int :dashboard_user_id
+      String :name
+      String :street_address, size: 1024
+      String :city
+      String :state
+      String :country
+      String :postal_code
+      String :district_name
+      String :district_city
+      String :district_state
+      String :district_zip
+      String :school_name
+      String :roles
```

Comment:
What values do we expect `roles` to take? A comment explaining the field would be helpful.